packages/legend-application-studio/style/components/editor/uml-editor/_uml-element-editor.scss

```diff
@@ -180,6 +180,26 @@
     }
   }
 
+  &__drag-div {
```

Comment:
We tend to name classes by their function, not by their implementation, so let's rename this to something like `drag-handler`, what if we change this to use `<span>` for some reason?

Reply:
makes sense! Changed